src/extensions/registries/page-menu-registry.ts

```diff
@@ -43,33 +37,23 @@ export class GlobalPageMenuRegistry extends BaseRegistry<PageMenuRegistration> {
   }
 }
 
-export class ClusterPageMenuRegistry extends BaseRegistry<ClusterPageMenuRegistration> {
-  @action
-  add(items: PageMenuRegistration[], ext: LensExtension) {
-    const normalizedItems = items.map(menuItem => {
-      menuItem.target = {
-        extensionId: ext.name,
-        ...(menuItem.target || {}),
-      };
-
-      return menuItem;
-    });
-
-    return super.add(normalizedItems);
-  }
-
+export class ClusterPageMenuRegistry extends PageMenuRegistry<ClusterPageMenuRegistration> {
   getRootItems() {
     return this.getItems().filter((item) => !item.parentId);
   }
 
   getSubItems(parent: ClusterPageMenuRegistration) {
-    return this.getItems().filter((item) => item.parentId === parent.id && item.target.extensionId === parent.target.extensionId);
+    return this.getItems().filter((item) => {
+      return item.parentId === parent.id && item.target.extensionId === parent.target.extensionId;
+    });
```

Comment:
Why the change? If you want to have them on separate lines then I would recommend the following instead:
```suggestion
    return this.getItems().filter((item) => (
      item.parentId === parent.id 
      && item.target.extensionId === parent.target.extensionId
    ));
```